src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -35,6 +38,9 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isGiverTeam = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
         isRecipientTeam = questionAttributes.recipientType.isTeam();
+        isRecipientOwnTeamMembers = questionAttributes.recipientType.isOwnTeamMembers();
+        isRecipientOtherTeams = questionAttributes.recipientType.isOtherTeams();
```

Comment:
same as above, we want to display message if recipient type is `Other teams` and there are no other teams in the course. If we take isTeam, it will include the students(who is giving feedback) team also.